cli/internal/cloudcmd/serviceaccount.go

```diff
@@ -0,0 +1,67 @@
+package cloudcmd
```

Comment:
```suggestion
/*
Copyright (c) Edgeless Systems GmbH

SPDX-License-Identifier: AGPL-3.0-only
*/

package cloudcmd
```

That part needs to stay ^^